services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -187,6 +188,12 @@ public void configure(Binder binder)
                     .setStatusFile(new File(taskAndStatusFile.get(1)))
             );
 
+            binder.bind(TaskReportFileWriter.class).toInstance(
+                new TaskReportFileWriter(
+                    new File(taskAndStatusFile.get(2))
```

Comment:
Should you update the [taskAndStatus arguments annotation](https://github.com/druid-io/druid/pull/5524/files#diff-b0cd94707fc9340ddc7eaf18fff275a7R117) to include this 3rd file? Also, maybe we should consider pulling these out into standalone variables at startup, like `taskFileName` `statusFileName` `reportFileName`, so it's more obvious what they are?

Reply:
Updated the annotation and made standalone variables for the file paths